R/1.0_module_data_summary.R

```diff
@@ -393,3 +393,5 @@ get_object_filter_overview_MultiAssayExperiment <- function(filtered_data, unfil
   experiment_info <- cbind(subset(experiment_obs_info, select = dataname:obs_filtered), experiment_subjects_info)
   rbind(mae_info, experiment_info)
 }
+
+globalVariables(c("obs", "obs_filtered", "subjects", "subjects_filtered", "colname"))
```

Comment:
Have you considered avoiding using `subset`?

The documentation suggests that it should only be used in interactive mode

![image](https://github.com/user-attachments/assets/f444a270-65bb-4d7e-8c3f-d9314977d48c)

Reply:
Thanks @averissimo for taking a look. I think we should also reconsider changing `transform()`  that we use into `object$ <-`

Reply:
Thanks @averissimo I got rid of NSE in this commit 114634ec8e8fccfd31fb04c3d77f8c817c746b1a
I think it's more R base-vanilla vibe right now

Reply:
Awesome!